doc/geany.txt

```diff
@@ -1645,14 +1642,14 @@ the format::
     name.lang_ext.tags
 
 *lang_ext* is one of the extensions set for the filetype associated
-with the tags. See the section called `Filetype extensions`_ for
+with the tags parser. See the section called `Filetype extensions`_ for
 more information.
 
 
 Default global tags files
 `````````````````````````
 
-For some languages, a list of global tags is loaded when the
+For some languages, global tags files are loaded when the
 corresponding filetype is first used. Currently these are for:
```

Comment:
This is a bit misleading, I guess this sentence is meant to say that we have some builtin tags files…